RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 Unreleased
 ------
 * [**] Add proper handling for single use blocks such as the more block [#3042]
+* [***] New Block: Audio
```

Comment:
@ceyhun Based on the other release note changes in this file the three-dot release note items seem to go above the two-dot ones. I am not sure if this has to be changed. LMK.

Reply:
Will update it 👍